tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/transaction/TransactionStartController.java

```diff
@@ -101,15 +115,9 @@ public String getValue() {
 
 					transactionSettings.add(transactionSetting);
 				}
-
 			}
-
 		});
 
-		if (isolationLevel[0] == null) {
-			throw new IllegalStateException("Isolation level must be specified");
```

Comment:
I took this check back out because it shouldn't be necessary: if no isolation level is supplied the connection should just use its default setting.